internal/util/update/update_test.go

```diff
@@ -24,13 +24,11 @@ import (
 	"github.com/GoogleContainerTools/kpt/internal/gitutil"
 	"github.com/GoogleContainerTools/kpt/internal/testutil"
 	"github.com/GoogleContainerTools/kpt/internal/testutil/pkgbuilder"
-	"github.com/GoogleContainerTools/kpt/internal/util/get"
+	testmgr "github.com/GoogleContainerTools/kpt/internal/util/testutil"
```

Comment:
I see testutil pkg already being there, any reason we can't move this there.

Also,

testutil.TestSetupManager{} reads better than testmgr.TestSetupManager :) if we are able to move it in testutil.

Reply:
Updated the PR by moving the `TestSetupManager` into `github.com/GoogleContainerTools/kpt/internal/testutil` and eliminating the package at `github.com/GoogleContainerTools/kpt/internal/util/testutil`.